docs/application/native/guides/multimedia/media-handle.md

```diff
@@ -7,11 +7,11 @@ The main features of the Media Tool API include:
 
 - Creating the media format handle
 
-  You can [set and get video or audio information](#format) using the `media_format_h` handle.
+  You can [set and get video or audio information](#format) using the `media_format_h` handle like average bps, bit, channel, channel mask, mime type, samplerate, width, height and frame rate for each type.(audio/video/text)
```

Comment:
Suggest rephrasing to:

You can [set and get video or audio information](#format) such as average bps, bit, channel, channel mask, mime type, sample rate, width, height, and frame rate for each type (audio, video, or text) using the `media_format_h` handle.

Please verify.
Also, is it sample rate or samplerate?

Reply:
Done.